processing/data_collection/gazette/spiders/pr_foz_do_iguacu.py

```diff
@@ -0,0 +1,71 @@
+import datetime as dt
+
+import scrapy
+from scrapy.http import FormRequest
+
+from gazette.items import Gazette
+
+
+class PrFozDoIguacuSpider(scrapy.Spider):
+    MUNICIPALITY_ID = '4108304'
+    name = 'pr_foz_do_iguacu'
+    allowed_domains = ['pmfi.pr.gov.br']
+    start_urls = ['http://www.pmfi.pr.gov.br/utilidades/diario/index.xhtml']
+    AVAILABLE_FROM = dt.date(2015, 1, 1)  # actually September/2007
```

Comment:
What about moving this contant to line 10?

Reply:
Using the BaseGazetteSpider, the constant wasn't needed anymore, tks